src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -397,7 +397,7 @@ public static bool IsGrainClass(Type type)
         public static bool IsSystemTargetClass(Type type)
         {
             Type systemTargetType;
-            if (!TryResolveType("Orleans.Runtime.SystemTarget, OrleansRuntime", out systemTargetType)) return false;
+            if (!TryResolveType("Orleans.Runtime.SystemTarget", out systemTargetType)) return false;
```

Comment:
What if we just moved this method to the test where it's used and then referenced `SystemTarget` directly? Like this:
![image](https://cloud.githubusercontent.com/assets/203839/20739510/66fa7caa-b70f-11e6-8783-5005e7dff075.png)